Phonebook.Frontend/src/styles/overrides.scss

```diff
@@ -22,6 +22,19 @@ $config: mat-typography-config();
   height: inherit;
 }
 
+// This fixes https://github.com/angular/components/issues/11094
+mat-card.mat-card{ //two times mat-card to override component styling of the material component (through higher specificity)
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+
+  .mat-card-content{
+    flex-grow: 1;
+    overflow: auto;
```

Comment:
remove if not necessary (does nothing for me), because it bugs other mat cards: 
![image](https://user-images.githubusercontent.com/13590797/67570050-c52c0c80-f730-11e9-8d81-0e626477c160.png)

Reply:
Still open